packages/components/src/pages/ChatSearch.vue

```diff
@@ -221,6 +221,15 @@ export default {
     },
   },
   methods: {
+    getAppMapState() {
+      return this.$refs.vappmap?.getState();
```

Comment:
FYI, I've noticed something's up with the optional chaining operator. It should transpile correctly into the distribution bundle, but it throws an error in Storybook.

Reply:
This is used in a method, so it should be fine.

Reply:
Ok - do I need to change it or not? :-)